applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -1533,34 +1436,43 @@ int unicast_client_discover(struct bt_conn *conn, enum unicast_discover_dir dir)
 	return ret;
 }
 
-int unicast_client_start(void)
+int unicast_client_start(enum bt_audio_dir dir)
 {
-	int ret_left = 0;
-	int ret_right = 0;
+	int ret;
+	int ret_total = 0;
 
-	if (le_audio_ep_state_check(headsets[AUDIO_CH_L].sink_stream.ep,
-				    BT_BAP_EP_STATE_QOS_CONFIGURED)) {
-		ret_left = bt_bap_stream_enable(&headsets[AUDIO_CH_L].sink_stream,
-						lc3_preset_sink.codec_cfg.meta,
-						lc3_preset_sink.codec_cfg.meta_len);
+	for (int i = 0; i < ARRAY_SIZE(headsets); i++) {
+		if (dir == BT_AUDIO_DIR_SINK) {
+			/* Start all streams in the configured state */
+			if (le_audio_ep_state_check(headsets[i].sink_stream.ep,
+						    BT_BAP_EP_STATE_QOS_CONFIGURED)) {
+				ret = bt_bap_stream_enable(&headsets[i].sink_stream,
+							   lc3_preset_sink.codec_cfg.meta,
+							   lc3_preset_sink.codec_cfg.meta_len);
 
-		if (ret_left) {
-			LOG_WRN("Failed to enable left stream: %d", ret_left);
-		}
-	}
+				if (ret) {
+					LOG_WRN("Failed to enable stream %d: %d", i, ret);
```

Comment:
Here we are at indentation level 5. Please refactor.

Reply:
Combine id_dir_sink and if ep_check